mappings/net/minecraft/util/CsvOutput.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_4456 net/minecraft/util/CsvOutput
```

Comment:
Maybe the class can be named something like `CsvWriter` (similar to how we have `java.io.Writer`)?